examples/testing-sc/README.md

```diff
@@ -0,0 +1,278 @@
+# Source to Knative Service
+
+**before you proceed**: the example in this directory illustrates the use of
+the latest components and functionality of Cartographer (including some that
+may not have been included in the latest release yet). Make sure to check out
+the version of this document in a tag that matches the latest version (for
+instance, https://github.com/vmware-tanzu/cartographer/tree/v0.0.7/examples).
+
+---
+
+The [basic-sc] example illustrates how an App Operator group could set up a software
+supply chain such that source code gets continuously built using the best
+practices from [buildpacks] via [kpack/Image] and deployed to the cluster using
+[knative-serving]. And the [runnable-tekton] example demonstrated how to make tekton tasks
+updatable. This example will bring these two together to insert a testing step into a supply chain.
+
+```
+
+  source --> tests --> image --> knative service
+
+```
+
+As with [basic-sc], the directories here are structured in a way to reflect which Kubernetes
+objects would be set by the different personas in the system:
+
+
+```
+  '── shared | testing-sc
+      ├── 00-cluster                         preconfigured cluster-wide objects
+      │                                        to configured systems other than
+      │                                              cartographer (like, kpack)
+      │
+      │
+      ├── app-operator                      cartographer-specific configuration
+      │   ├── supply-chain-templates.yaml            that an app-operator would
+      │   ├── ...                                                        submit
+      │   └── supply-chain.yaml
+      │
+      │
+      └── developer                         cartographer-specific configuration
+          ├── ...                                       that an app-dev submits
+          └── workload.yaml
+```
+
+## Prerequisites
+
+1. Install the prerequisites of the [basic-sc] example.
+
+2. Install [tekton]
+
+```bash
+TEKTON_VERSION=0.30.0 kubectl apply -f https://storage.googleapis.com/tekton-releases/pipeline/previous/v$TEKTON_VERSION/release.yaml
+```
+
+
+## Running the example in this directory
+
+As with [basic-sc], this example uses two directories with sub-directories of kubernetes resources:
+[../shared](../shared) and [.](.).
+
+The shared directory has a subdirectory for cluster-wide configuration: [./00-cluster](./00-cluster)
+
+There is a subdirectory of cartographer-specific files that an App Operator would submit
+in both this and in the shared directory:
+[./app-operator](./app-operator) and [../shared/app-operator](../shared/app-operator)
+
+Finally, in this and in the shared directory there is a subdirectory containing
+Kubernetes objects that a developer would submit: [./developer](./developer) and
+[../shared/developer](../shared/developer)
+
+### Configuring the example
+
+Follow the [example configuration steps in basic-sc](../basic-sc/README.md#configuring-the-example)
```

Comment:
It might be worth pointing to configuring in basic-sc, but then having the ytt command here.

Reply:
Addressed in the follow on story: https://github.com/vmware-tanzu/cartographer/pull/504